SparkPost/2web2ui
# AC-711: Update 'Email' role to 'Templates'

User feedback shows that `email` is confusing for the role instead of actual intended functions of role. The role will likely be changed API side eventually to reflect changes.

### What Changed
 - User List Page now shoes 'templates' instead of 'email' role
 - RoleRadioGroup (found in User edit form and User invite) now shows 'Templates' instead of 'Email'
 - Pulled formatter to helper function

### How To Test
 - Check user invite form says `Templates` as role
 - Check user edit form says `Templates` instead of `email` as role
 - Check list page shows templates for new templates user or older `email` users

### To Do
